docs/markdown/line-series.md

```diff
@@ -78,6 +78,11 @@ An object which holds CSS properties that will be applied to the SVG element(s)
 
 ### Interaction handlers
 
+Note - interacting with a line may be difficult especially with the standard width. To address that, consider:
```

Comment:
Plus one

Reply:
1) do i need right click interaction for sankey and treemap - maybe, but they don't follow the onValueClick / onSeriesClick model to begin with. So that suggests this should be a separate effort. 
2) opportunity for more tests, I agree but again I think there's need for something more strategic than bundled with this specific PR. also at the time of this PR there were 999 tests for the record
3) documentation super non dry - what i'm thinking is moving documentation content from flat markdown file to a more database like structure so we could render doc pages based on props and have a finer control of look and feel for what's a series prop description vs what's explanatory text. right now, everything is a title or not, bold or not, italics or not and that's it. ironic for front end people.